mlflow/h2o.py

```diff
@@ -97,6 +121,18 @@ def load_pyfunc(path):
     :param path: Local filesystem path to the model saved by :py:func:`mlflow.h2o.save_model`.
     :rtype: Pyfunc format model with function
             ``model.predict(pandas DataFrame) -> pandas DataFrame``.
+
+    >>> import mlflow
+    >>> import mlflow.h2o
+    >>> import h2o
+    >>> # set the path to where the h2o model is saved: local or remote, accessible
+    >>> # from code here
+    >>> h2o.init()
+    >>> h2o_model_dir = ...
+    >>> # set the test Pandas DataFrame
+    >>> pandas_df = ...
+    >>> h2o_model = mlflow.h2o.load_pyfunc(h2o_model_dir)
+    >>> predictions = h2o_model.predict(pandas_df)
```

Comment:
I think this is a bit confusing in the code.  

The intended way of loading a pyfunc model is to call mlflow.pyfunc.load_pyfunc() which will read the MLmodel yaml file and call the appropriate method to load the model, in this case mlflow.h2o.load_pyfunc(). The important difference is that mlflow.pyfunc.load_pyfunc expects a path to a top level directory containing MLmodel file while the mlflow.h2o.load_pyfunc expects a path to flavor specific data folder. So I think the above code snippet is incorrect.

I don't think we need and example for this function. Instead, we should clarify that it is intended to be used internally.

Reply:
Now, I'm confused because the API signature for load_pyfunc(path) expects a "path" where model was saved from 'save_model(...) call. Will stop by later today.